modules/sap_emptyha_svpc/README.md

```diff
@@ -0,0 +1,220 @@
+
+# SAP EMPTY HA SVPC Submodule
+
+This module deals with SAP EMPTY HA configuration and deployment.
+
+## Usage
+
+The resources/services/activations/deletions that this module will create/trigger are:
+
+- Create Primary and Secondary Compute Instance that will host SAP Empty HA.
+- Create a Static IP Address for the two Compute Instance's.
+- Create 2 Persistent Disks to host SAP Empty HA's File systems on primary and secondary nodes.
+
+You can go in the [examples](../../examples) folder complete working example. However, here's an example of how to use the module in a main.tf file.
+
+```hcl
+provider "google" {
+  version = "~> 2.6.0"
+  region  = "${var.region}"
+}
+
+module "gcp_sap_emptyha_svpc" {
+source                      = "terraform-google-modules/sap/google/modules/sap_emptyha_svpc"
+post_deployment_script   = "${var.post_deployment_script}"
+subnetwork               = "${var.subnetwork}"
+linux_image_family       = "${var.linux_image_family}"
+linux_image_project      = "${var.linux_image_project}"
+instance_type            = "${var.instance_type}"
+network_tags             = "${var.network_tags}"
+project_id               = "${var.project_id}"
+subnetwork_project       = "${var.subnetwork_project}"
+region                   = "${var.region}"
+service_account_email    = "${var.service_account_email}"
+boot_disk_size           = "${var.boot_disk_size}"
+boot_disk_type           = "${var.boot_disk_type}"
+autodelete_disk          = "true"
+sap_deployment_debug     = "false"
+primary_instance_name    = "${var.primary_instance_name}"
+secondary_instance_name  = "${var.secondary_instance_name}"
+primary_zone             = "${var.primary_zone}"
+secondary_zone           = "${var.secondary_zone}"
+sap_vip                  = "${var.sap_vip}"
+sap_vip_secondary_range  = "${var.sap_vip_secondary_range}"
+primary_instance_ip      = "${var.primary_instance_ip}"
+secondary_instance_ip    = "${var.secondary_instance_ip}"
+public_ip                = "${var.public_ip}"
+ip_cidr_range            = "${var.ip_cidr_range}"
+sap_vip_internal_address = "${var.sap_vip_internal_address}"
+  }
+
+```
+## Requirements
+
+
+# Use Case 1: When only private Ip is the need , with no secondary ip ranges in subnetwork .
```

Comment:
Users should not be required to modify the module code in order to use it. If there are incompatible use cases then they need to be split in to separate submodules.

Reply:
removed these comments